address.json

```diff
@@ -14,6 +14,45 @@
 	"claimpolka.live": [
 		"12aCLUsyTCWn5Dzf5wxuuRm93dx9rkAF5rqw9zBDTV5NSY1E"
 	],
+	"polkadotevent2022.net": [
+		"1Rb2Uhkkztfz2oJewKXcEBRykQurtJAZ1zLEsWyrFs24RFD"
+	],
+	"polka2022.net": [
+		"1Rb2Uhkkztfz2oJewKXcEBRykQurtJAZ1zLEsWyrFs24RFD"
+	],
```

Comment:
It doesn't hurt adding it all-over, but a single is enough. The name is just used as a hint when the address pops up, so only one will be used.

Reply:
Understood, thanks, sorry for the overhead.

Reply:
Less work when an address is known :)